samples/dashboard/demos/component-html/demo.js

```diff
@@ -0,0 +1,23 @@
+const dashboard = new Dashboard.Dashboard('container', {
+    gui: {
+        enabled: true,
+        layouts: [{
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-4'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        type: 'html',
```

Comment:
Can you add two cells with two different HTML  so it looks more like a dashboard?
Perhaps one with a static image?